src/visualizer/sortingvisualizer/util/GeneralUtil.js

```diff
@@ -9,8 +9,8 @@ import { generateMergeSortArray } from './MergeSortUtil';
  * Resets the array given.
  *
  * @param {string} algo Algorithm used.
- * @param {Object[]} arr Array to be reset.
- * @returns {Object[]} Array that has been reset.
+ * @param {object[]} arr Array to be reset.
+ * @returns {object[]} Array that has been reset.
```

Comment:
I think lower-cased is more correct for `Object`?

![image](https://user-images.githubusercontent.com/65328119/103145180-b09e3680-4770-11eb-87ae-42fa21d6b626.png)